lean/src/valid/mathd_algebra_140.lean

```diff
@@ -5,7 +5,11 @@ Authors: Kunhao Zheng
 -/
 import data.real.basic
 
-theorem mathd_algebra_140(a b c : ℝ) (h₀ : 0 < a ∧ 0 < b ∧ 0 < c) (h₁ : ∀ x, 24 * x ^ 2 - 19 * x - 35 = ( ( ( a * x ) - 5 ) * ( ( 2 * ( b * x ) ) + c ) ) ) : a * b - 3 * c = -9 :=
+theorem mathd_algebra_140
+  (a b c : ℝ)
+  (h₀ : 0 < a ∧ 0 < b ∧ 0 < c)
+  (h₁ : ∀ x, 24 * x ^ 2 - 19 * x - 35 = ( ( ( a * x ) - 5 ) * ( ( 2 * ( b * x ) ) + c ) ) ) :
```

Comment:
I've also tried to remove spaces next to parenthesis and spaces around `^` to improve compactness